src/RouterRoot.js

```diff
@@ -21,7 +21,22 @@ import Navbar from "./components/navbar/Navbar";
 import { firebaseAuth } from './firebase';
 import AnswerWaitTimerContainer from './components/answer-wait-timer/AnswerWaitTimerContainer';
 
+export const QuizContext = React.createContext();
 
+export class QuizContextProvider extends React.Component{
+    render(){
+        return(
+            <QuizContext.Provider value={{
+                state: this.state,
```

Comment:
why are we passing the entire state? we should pass only items which we need.

Reply:
Makes sense, but anyways we would be needing everything that's present in the state.